example/index.js

```diff
@@ -2,38 +2,109 @@ const {
   ClientService,
 } = require('..');
 
+const path = require('path');
+
 (async () => {
   const properties = {
     'scalar.ledger.client.server_host': 'localhost',
```

Comment:
Sorry, the config namespace and some of the names are updated.
Please refer to this.
https://github.com/scalar-labs/scalar-samples/blob/master/conf/client.properties

Reply:
Because the constructor of ClientService parses the properties.

Reply:
Sorry I didn't notice this.
Let me fix it.